Clinical-Genomics/scout
# Skip ClinVar variant submission for variants that are not loaded

fix #2180 
The bug is caused by a function trying to submit to clinVar variants that are not loaded (variants that were once loaded for the case but disappeared after the case was re-runned).

**How to test on your local computer**:
-  On master branch.
-  Go to demo case and pin your favorite variant (POT1?).
-  Using mongodb compass, edit the case document and add some fake old variant to the list of the pinned variants. Like this:

![image](https://user-images.githubusercontent.com/28093618/98125805-25fa3300-1eb5-11eb-9428-c1be210b327f.png)

The case pinned variants will look like this:
![image](https://user-images.githubusercontent.com/28093618/98125933-4a560f80-1eb5-11eb-8c15-9a26ef98fc1f.png)

-  Add one phenotype term (HPO term) to the case so that it will be possible to create a clinVar submission.
-  Go back to the pinned variant (the existing one) and click the button `Submit to ClinVar`
-  Observe that the page crashed with the same error in #2180 
- Switch to this branch and repeat the last action (you might need to refresh the variant page)
- The ClinVar export shoud work, showing only the existing variant.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
